python/cuml/ensemble/randomforestclassifier.pyx

```diff
@@ -232,7 +232,7 @@ class RandomForestClassifier(BaseRandomForestModel,
         .. deprecated:: 0.19
            Parameter 'quantile_per_tree' is deprecated and will be removed in
            subsequent release.
-    use_experimental_backend : boolean (default = False)
+    use_experimental_backend : boolean (default = True)
```

Comment:
Can we add a little to the comment to the effect that this backend is still experimental because its impact on regression is still being evaluated but it should be considered stable for classification?
(I don't think we actually need to change the parameter name until 0.20, but a little note would help)

Reply:
Done